aodndata/aims/faimms.py

```diff
@@ -52,20 +52,18 @@ def get_faimms_site_name(filepath):
         site_code = netcdf_file_obj.site_code
         netcdf_file_obj.close()
 
-        if 'DAV' in site_code:
-            return 'Davies_Reef'
-        elif 'OI' in site_code:
-            return 'Orpheus_Island'
-        elif 'OTI' in site_code:
-            return 'One_Tree_Island'
-        elif 'RIB' in site_code:
-            return 'Rib_Reef'
-        elif 'MRY' in site_code or 'MYR' in site_code:
-            return 'Myrmidon_Reef'
-        elif 'LIZ' in site_code:
-            return 'Lizard_Island'
-        elif 'HI' in site_code:
-            return 'Heron_Island'
+        site_codes = {'DAV': 'Davies_Reef',
+                      'OI': 'Orpheus_Island',
+                      'OTI': 'One_Tree_Island',
+                      'RIB': 'Rib_Reef',
+                      'MRY': 'Myrmidon_Reef',
+                      'LIZ': 'Lizard_Island',
+                      'HI': 'Heron_Island'}
+
+        main_site_code = [site for site in site_codes.keys() if site in site_code]
```

Comment:
This could just be:

```python
return site_codes.get(site_code, None)
```
You should always explicitly return something. Also, if you allow this to return None, then the other code has to handle it. Should this raise an Exception if the site code is invalid?

Reply:
for example 
```python
site_code = 'HIRP4'
```
this bit of code tests that "HI" from site_code is contained somewhere in the dictionarry site_codes

so  this would not work
 ```python
 site_codes.get(site_code, None)
```

the exception is raised in this other function
```python
def get_main_faimms_site_name_path(self, filepath):
```
but I guess you're right. It should be moved